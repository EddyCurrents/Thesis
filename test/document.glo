\glossaryentry{ASIC?\glossentry{asic}|setentrycounter[]{page}\glsnumberformat}{1}
