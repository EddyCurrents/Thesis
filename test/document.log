This is pdfTeX, Version 3.14159265-2.6-1.40.16 (MiKTeX 2.9) (preloaded format=pdflatex 2015.9.11)  23 SEP 2015 12:31
entering extended mode
**document.tex
(document.tex
LaTeX2e <2015/01/01> patch level 2
Babel <3.9m> and hyphenation patterns for 69 languages loaded.
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\base\article.cls"
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\base\size12.clo"
File: size12.clo 2014/09/29 v1.4h Standard LaTeX file (size option)
)
\c@part=\count79
\c@section=\count80
\c@subsection=\count81
\c@subsubsection=\count82
\c@paragraph=\count83
\c@subparagraph=\count84
\c@figure=\count85
\c@table=\count86
\abovecaptionskip=\skip41
\belowcaptionskip=\skip42
\bibindent=\dimen102
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\glossaries\base\glossaries.sty"
Package: glossaries 2015/09/09 v4.18 (NLCT)

("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\base\ifthen.sty"
Package: ifthen 2014/09/29 v1.1c Standard LaTeX ifthen package (DPC)
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\xkeyval\xkeyval.sty"
Package: xkeyval 2014/12/03 v2.7a package option processing (HA)

("C:\Program Files (x86)\MiKTeX 2.9\tex\generic\xkeyval\xkeyval.tex"
("C:\Program Files (x86)\MiKTeX 2.9\tex\generic\xkeyval\xkvutils.tex"
\XKV@toks=\toks14
\XKV@tempa@toks=\toks15

("C:\Program Files (x86)\MiKTeX 2.9\tex\generic\xkeyval\keyval.tex"))
\XKV@depth=\count87
File: xkeyval.tex 2014/12/03 v2.7a key=value parser (HA)
))
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\mfirstuc\mfirstuc.sty"
Package: mfirstuc 2015/09/09 v2.0 (NLCT)

("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\etoolbox\etoolbox.sty"
Package: etoolbox 2015/08/02 v2.2a e-TeX tools for LaTeX (JAW)
\etb@tempcnta=\count88
)
\@glsmfirst=\toks16
\@glsmrest=\toks17
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\textcase\textcase.sty"
Package: textcase 2004/10/07 v0.07 Text only upper/lower case changing (DPC)
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\xfor\xfor.sty"
Package: xfor 2009/02/05 v1.05 (NLCT)
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\datatool\datatool-base.sty"
Package: datatool-base 2015/07/11 v2.23 (NLCT)

("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\amsmath\amsmath.sty"
Package: amsmath 2013/01/14 v2.14 AMS math features
\@mathmargin=\skip43

For additional information on amsmath, use the `?' option.
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\amsmath\amstext.sty"
Package: amstext 2000/06/29 v2.01

("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\amsmath\amsgen.sty"
File: amsgen.sty 1999/11/30 v2.0
\@emptytoks=\toks18
\ex@=\dimen103
))
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\amsmath\amsbsy.sty"
Package: amsbsy 1999/11/29 v1.2d
\pmbraise@=\dimen104
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\amsmath\amsopn.sty"
Package: amsopn 1999/12/14 v2.01 operator names
)
\inf@bad=\count89
LaTeX Info: Redefining \frac on input line 210.
\uproot@=\count90
\leftroot@=\count91
LaTeX Info: Redefining \overline on input line 306.
\classnum@=\count92
\DOTSCASE@=\count93
LaTeX Info: Redefining \ldots on input line 378.
LaTeX Info: Redefining \dots on input line 381.
LaTeX Info: Redefining \cdots on input line 466.
\Mathstrutbox@=\box26
\strutbox@=\box27
\big@size=\dimen105
LaTeX Font Info:    Redeclaring font encoding OML on input line 566.
LaTeX Font Info:    Redeclaring font encoding OMS on input line 567.
\macc@depth=\count94
\c@MaxMatrixCols=\count95
\dotsspace@=\muskip10
\c@parentequation=\count96
\dspbrk@lvl=\count97
\tag@help=\toks19
\row@=\count98
\column@=\count99
\maxfields@=\count100
\andhelp@=\toks20
\eqnshift@=\dimen106
\alignsep@=\dimen107
\tagshift@=\dimen108
\tagwidth@=\dimen109
\totwidth@=\dimen110
\lineht@=\dimen111
\@envbody=\toks21
\multlinegap=\skip44
\multlinetaggap=\skip45
\mathdisplay@stack=\toks22
LaTeX Info: Redefining \[ on input line 2665.
LaTeX Info: Redefining \] on input line 2666.
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\substr\substr.sty"
Package: substr 2009/10/20 v1.2 Handle substrings
\c@su@anzahl=\count101
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\datatool\datatool-fp.sty"
Package: datatool-fp 2015/07/11 v2.23 (NLCT)

("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp.sty"
Package: fp 1995/04/02

`Fixed Point Package', Version 0.8, April 2, 1995 (C) Michael Mehlich
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\defpattern.sty"
Package: defpattern 1994/10/12
\actioncount=\count102
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-basic.sty"
Package: fp-basic 1996/05/13
\FP@xs=\count103
\FP@xia=\count104
\FP@xib=\count105
\FP@xfa=\count106
\FP@xfb=\count107
\FP@rega=\count108
\FP@regb=\count109
\FP@regs=\count110
\FP@times=\count111
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-addons.sty"
Package: fp-addons 1995/03/15
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-snap.sty"
Package: fp-snap 1995/04/05
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-exp.sty"
Package: fp-exp 1995/04/03
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-trigo.sty"
Package: fp-trigo 1995/04/14
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-pas.sty"
Package: fp-pas 1994/08/29
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-random.sty"
Package: fp-random 1995/02/23
\FPseed=\count112
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-eqn.sty"
Package: fp-eqn 1995/04/03
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-upn.sty"
Package: fp-upn 1996/10/21
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\fp\fp-eval.sty"
Package: fp-eval 1995/04/03
)))
\@dtl@toks=\toks23
\@dtl@tmpcount=\count113
\dtl@tmplength=\skip46
\dtl@sortresult=\count114
\@dtl@numgrpsepcount=\count115
\@dtl@datatype=\count116
\dtl@codeA=\count117
\dtl@codeB=\count118
\@dtl@foreach@level=\count119
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\glossaries\base\glossaries-compat
ible-307.sty"
Package: glossaries-compatible-307 2015/09/09 v4.18 (NLCT)
)
\gls@level=\count120
\@gls@tmpb=\toks24
\gls@tmplen=\skip47
\glskeylisttok=\toks25
\glslabeltok=\toks26
\glsshorttok=\toks27
\glslongtok=\toks28

("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\glossaries\styles\glossary-hypern
av.sty"
Package: glossary-hypernav 2015/09/09 v4.18 (NLCT)
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\glossaries\styles\glossary-list.s
ty"
Package: glossary-list 2015/09/09 v4.18 (NLCT)
\glslistdottedwidth=\skip48
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\glossaries\styles\glossary-long.s
ty"
Package: glossary-long 2015/09/09 v4.18 (NLCT)
 ("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\tools\longtable.sty"
Package: longtable 2014/10/28 v4.11 Multi-page Table package (DPC)
\LTleft=\skip49
\LTright=\skip50
\LTpre=\skip51
\LTpost=\skip52
\LTchunksize=\count121
\LTcapwidth=\dimen112
\LT@head=\box28
\LT@firsthead=\box29
\LT@foot=\box30
\LT@lastfoot=\box31
\LT@cols=\count122
\LT@rows=\count123
\c@LT@tables=\count124
\c@LT@chunks=\count125
\LT@p@ftn=\toks29
)
\glsdescwidth=\skip53
\glspagelistwidth=\skip54
)
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\glossaries\styles\glossary-super.
sty"
Package: glossary-super 2015/09/09 v4.18 (NLCT)
 ("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\supertabular\supertabular.sty"
Package: supertabular 2004/02/20 v4.1e the supertabular environment
\c@tracingst=\count126
\ST@wd=\dimen113
\ST@rightskip=\skip55
\ST@leftskip=\skip56
\ST@parfillskip=\skip57
\ST@pageleft=\dimen114
\ST@headht=\dimen115
\ST@tailht=\dimen116
\ST@pagesofar=\dimen117
\ST@pboxht=\dimen118
\ST@lineht=\dimen119
\ST@stretchht=\dimen120
\ST@prevht=\dimen121
\ST@toadd=\dimen122
\ST@dimen=\dimen123
\ST@pbox=\box32
))
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\glossaries\styles\glossary-tree.s
ty"
Package: glossary-tree 2015/09/09 v4.18 (NLCT)
\glstreeindent=\skip58
))
\glswrite=\write3
\glo@main@file=\write4
Package glossaries Info: Writing glossary file document.glo on input line 6.
 (document.aux)
LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 12.
LaTeX Font Info:    ... okay on input line 12.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 12.
LaTeX Font Info:    ... okay on input line 12.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 12.
LaTeX Font Info:    ... okay on input line 12.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 12.
LaTeX Font Info:    ... okay on input line 12.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 12.
LaTeX Font Info:    ... okay on input line 12.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 12.
LaTeX Font Info:    ... okay on input line 12.

No file document.gls.
[1

{C:/Users/Ã˜yvind/AppData/Local/MiKTeX/2.9/pdftex/config/pdftex.map}]
(document.aux) ) 
Here is how much of TeX's memory you used:
 4577 strings out of 493673
 65950 string characters out of 3141524
 174451 words of memory out of 3000000
 7877 multiletter control sequences out of 15000+200000
 4544 words of font info for 17 fonts, out of 3000000 for 9000
 1025 hyphenation exceptions out of 8191
 46i,4n,39p,1910b,113s stack positions out of 5000i,500n,10000p,200000b,50000s
<C:/Program Files (x86)/MiKTeX 2.9/fonts/type1/public/amsfonts/
cm/cmbx12.pfb><C:/Program Files (x86)/MiKTeX 2.9/fonts/type1/public/amsfonts/cm
/cmr12.pfb>
Output written on document.pdf (1 page, 25878 bytes).
PDF statistics:
 14 PDF objects out of 1000 (max. 8388607)
 0 named destinations out of 1000 (max. 500000)
 1 words of extra memory for PDF output out of 10000 (max. 10000000)

