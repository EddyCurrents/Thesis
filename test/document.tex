% Created 2015-09-02
\documentclass[12pt]{article}

%\usepackage{../mystyle}
\usepackage{glossaries}
\makeglossaries

\newacronym{asic}{ASIC}{Application-Specific Integrated Circuit}
%\usepackage{../myglossary}
%\graphicspath{{./../img/}}

\begin{document}

\section*{Acknowledgements}
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec a diam lectus. Sed sit amet ipsum mauris. Maecenas congue ligula ac quam viverra nec consectetur ante hendrerit. Donec et mollis dolor. \gls{asic} Praesent et diam eget libero egestas mattis sit amet vitae augue. Nam tincidunt congue enim, ut porta lorem lacinia consectetur. Donec ut libero sed arcu vehicula ultricies a non tortor. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean ut gravida lorem.
%\addcontentsline{toc}{section}{Acknowledgements}

%\blankpage

%\printglossary[type=acronym]
\printglossaries
%\addcontentsline{toc}{section}{Glossary}

%\blankpage
%
%\listoffigures
%\addcontentsline{toc}{section}{List of Figures}
%
%\blankpage
%
%\listoftables
%\addcontentsline{toc}{section}{List of Tables}
%
%\blankpage
%
%\lstlistoflistings
%\addcontentsline{toc}{section}{List of Listings}

\end{document}
