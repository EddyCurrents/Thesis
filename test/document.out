\BOOKMARK [1][-]{Doc-Start}{Acknowledgements}{}% 1
\BOOKMARK [1][-]{Doc-Start}{Glossary}{}% 2
